### PR TITLE
Convert to ES6

### LINKED ISSUE #79: CoffeeScript -> ES6
Would you guys be up for moving from CoffeeScript to ES6 for this package? I can take a crack at it in the coming weeks if so.

### PR DESCRIPTION
**Pull request by YurySolovyov**
This aims to fix #79.
I ran it though [`decaffeinate-project.org`](http://decaffeinate-project.org/) and fixed some places that looked obvious.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-  "main": "./lib/asar.js",
+  "main": "./src/asar.js",
   "name": "asar",
   "description": "Creating Electron app packages",
   "version": "0.12.3",
   "bin": {
     "asar": "bin/asar"
   },
+  "engines": {
+    "node": ">=4.6"
```
Comment (YurySolovyov) [inline package.json]: Latest LTS at the time, I think.

**Comment by YurySolovyov** [comment]
Sure, will do

**Comment by YurySolovyov** [comment]
@kevinsawicki I ran it through, though it does not recognize some of the spec method names, not sure what to do here cause I'm not yet familiar with how standard works.

Also, not sure what's the best way to resolve conflicts here, "just" port recent changes?

**Comment by kevinsawicki** [comment]
>  I ran it through, though it does not recognize some of the spec method names, not sure what to do here cause I'm not yet familiar with how standard works.

Cloning this now and will see.

> Also, not sure what's the best way to resolve conflicts here, "just" port recent changes?

Sorry about these, will take a look.

**Comment by kevinsawicki** [comment]
Conflicts resolved and standard errors fixed

**Comment by kevinsawicki** [comment]
Will merge this next week after 0.12 LTS is expired 👍

**Comment by YurySolovyov** [comment]
Awesome, good to know

**Comment by YurySolovyov** [comment]
This should be good to merge now.

**Comment by kevinsawicki** [comment]
Thanks for this @YurySolovyov 🚢 

Will release shortly in `0.13.0`.

**Pull request merged**